sources/assets/Stride.Core.Assets.CompilerApp/PackageBuilder.cs

```diff
@@ -195,15 +195,15 @@ private void SaveBuildUpToDateFile(string msbuildUpToDateCheckFileBase, string p
             {
                 // Note: check if file exists (since it could be an "implicit package" from csproj)
                 if (File.Exists(package.FullPath))
-                    inputs.Add(package.FullPath.ToWindowsPath());
+                    inputs.Add(package.FullPath.ToOSPath());
 
                 // TODO: optimization: for nuget packages, directly use sha512 file rather than individual assets for faster checking
 
                 // List assets
                 foreach (var assetFolder in package.AssetFolders)
                 {
                     if (Directory.Exists(assetFolder.Path))
-                        inputs.Add(assetFolder.Path.ToWindowsPath() + @"\**\*.*");
+                        inputs.Add(new UFile(assetFolder.Path + @"\**\*.*").ToOSPath());
```

Comment:
Looks like the windows test failed, probably from this particular test ?

Reply:
Looks like path concatenation was incorrect (path from test: "../sources/engine/Stride.Shaders.Tests/GameAssets\**\*.*" - incorrect slashes). I adjusted code to replace wrong slashes. Hope It didnt look too hacky 😬